revise vignettes (#114)
ropensci/opencage: README.md

# opencage

<!-- badges: start -->

[![CRAN
Status](http://www.r-pkg.org/badges/version/opencage)](http://cran.r-project.org/package=opencage)
[![CRAN
Checks](https://cranchecks.info/badges/worst/opencage)](https://cran.r-project.org/web/checks/check_results_opencage.html)
[![CRAN Downloads per
Month](https://cranlogs.r-pkg.org/badges/opencage)](http://cran.r-project.org/package=opencage)
[![Travis Build
Status](https://travis-ci.org/ropensci/opencage.svg?branch=master)](https://travis-ci.org/ropensci/opencage)
[![AppVeyor Build
Status](https://ci.appveyor.com/api/projects/status/github/ropensci/opencage?branch=master&svg=true)](https://ci.appveyor.com/project/maelle/opencage)
[![codecov.io](https://codecov.io/github/ropensci/opencage/coverage.svg?branch=master)](https://codecov.io/github/ropensci/opencage?branch=master)
[![rOpenSci
Peer-Review](https://badges.ropensci.org/36_status.svg)](https://github.com/ropensci/software-review/issues/36)
[![License](https://img.shields.io/cran/l/opencage)](https://opensource.org/licenses/gpl-license)

<!-- badges: end -->

Geocode with the [OpenCage](https://opencagedata.com/) API, either from

> placename to longitude and latitude (forward geocoding) or from

Reviewer: ```suggestion
place name to longitude and latitude (forward geocoding) or from
```

> longitude and latitude to placename (reverse geocoding).

Reviewer: ```suggestion
longitude and latitude to the location name (reverse geocoding).
```

remotes::install_github("ropensci/opencage")
```

## Quickstart

For the best experience, we recommend that you read through the “Get
started” vignette (`vignette("intro_opencage")`), but if you are in a
hurry:

1.  Register at
    [opencagedata.com/users/sign\_up](https://opencagedata.com/users/sign_up).
2.  Generate an API key at the [OpenCage
    dashboard](https://opencagedata.com/dashboard#api-keys).
3.  Save your API key as an [environment
    variable](https://rstats.wtf/r-startup.html#renviron) like

>     `OPENCAGE_KEY=yourkey` in `.Renviron`. (See `help(oc_config)` for

Reviewer: ```suggestion
    `OPENCAGE_KEY=yourkey` in `.Renviron`. See `help(oc_config)` for
```

>     alternative ways to set your OpenCage API key.)

Reviewer: ```suggestion
    alternative ways to set your OpenCage API key.
```

> Now you are ready to turn placenames into latitude/longitude

Reviewer: ```suggestion
Now you are ready to turn place names into latitude and longitude
```

coordinates:

``` r
library(opencage)
oc_forward_df(placename = "Sarzeau")
```

    ## # A tibble: 1 x 4
    ##   placename oc_lat oc_lng oc_formatted         
    ##   <chr>      <dbl>  <dbl> <chr>                
    ## 1 Sarzeau     47.5  -2.76 56370 Sarzeau, France

> Or turn coordinates into placenames:

Reviewer: ```suggestion
Or turn a set of coordinates into the name of the location:
```

plus a whole lot more besides. Refer to the current full [list of
credits](https://opencagedata.com/credits).

## Meta

  - Please [report any issues or
    bugs](https://github.com/ropensci/opencage/issues).
  - License: GPL \>= 2

>   - Get citation information for `opencage` in R doing `citation(package

Reviewer: ```suggestion
  - Get citation information for `opencage` in R through `citation(package
```
